logp/core_test.go

```diff
@@ -46,9 +46,11 @@ func TestLogger(t *testing.T) {
 		log.Infow("some message with struct value", "metrics", someStruct)
 	}
 
-	TestingSetup()
+	err := TestingSetup()
+	assert.NoError(t, err)
 	exerciseLogger()
-	TestingSetup()
+	err = TestingSetup()
+	assert.NoError(t, err)
 	exerciseLogger()
```

Comment:
I believe this should be `require` instead of `assert`. Does it make sense to continue the test if `TestingSetup` fails?